### ISSUE TITLE
Where are the exercises?

### ISSUE THREAD

**Issue by Exagram** [donnemartin/system-design-primer#789]
I see https://github.com/donnemartin/system-design-primer#system-design-interview-questions-with-solutions has a table of exercises and solutions. However when I click on the Pastebin link I am redirected to https://github.com/donnemartin/system-design-primer/blob/master/solutions/system_design/pastebin/README.md which is the full solution - not the exercise.

Am I missing something?

**Comment by notdev-9** [comment]
No, the questions themselves are the exercises. Your job is to come up with the whole solutions for these design questions

**Comment by Exagram** [comment]
> No, the questions themselves are the exercises. Your job is to come up with the whole solutions for these design questions

My point is the first resource has a table full of "Solution" links but usually one would expect to see "Problem" links.

The second resource has the high-level design written out. Would it be better to make visible information that the interviewer would say out loud and hide the high-level design with **SPOILERS** accordion widgets so that readers can compare and contrast with their own original designs?

Do most interviewers usually expect both A) high-level designs and B) whole solutions (implemented in X programming language of choice)?

**Comment by ippbxusa** [comment]
Discover how [Postartica ](https://postartica.com/)champions your privacy with a secure, user-centric approach, adhering to GDPR and CCPA standards in their digital world.
